tests/grpc/historical.go

```diff
@@ -33,7 +33,7 @@ func MedianCheck(umee client.Client) error {
 	}
 
 	var exchangeRates sdk.DecCoins
-	for i := 0; i < 20; i++ {
+	for i := 0; i < 50; i++ {
```

Comment:
this was giving lots of issues when lowering block times, looks like it's solved this way